main.js

```diff
@@ -29,6 +29,9 @@ function main(args) {
         .usage('[options]')
         .option('-l, --listen <listen>', 'Host and port on which server listens (also node\'s identity in cluster)')
         .option('-h, --hosts <hosts>', 'Seed file of list of hosts to join')
+        .option('--suspect [suspect]', 'Suspect period in seconds')
```

Comment:
you should use `--suspect  <suspect>` to make a value required if `--suspect` is specified.